docs/content/latest/quick-start/build-apps/go/ysql-pg-ssl.md

```diff
@@ -0,0 +1,335 @@
+---
+title: Build a Go application that uses YSQL over SSL
+headerTitle: Build a Go application
+linkTitle: Go
+description: Build a sample Go application with the Go PostgreSQL driver and perform basic database operations with YugabyteDB with SSL/TLS.
+menu:
+  latest:
+    parent: build-apps
+    name: Go
+    identifier: go-4
+    weight: 552
+type: page
+isTocNested: true
+showAsideToc: true
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li>
+    <a href="../ysql-pgx-ssl/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - PGX - SSL
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-pq/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - PQ
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-pq-ssl/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - PQ - SSL
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-pg-ssl/" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - PG - SSL
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-gorm/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - GORM
+    </a>
+  </li>
+  <li>
+    <a href="../ycql/" class="nav-link">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL
+    </a>
+  </li>
+</ul>
+
+The following tutorial creates a simple Go application that connects to a YugabyteDB cluster using the [Go-pg client and ORM](https://pkg.go.dev/github.com/go-pg/pg), performs a few basic database operations — creating a table, inserting data, and running a SQL query — and then prints the results to the screen.
+
+## Prerequisites
+
+This tutorial assumes that:
+
+- YugabyteDB is up and running. If you are new to YugabyteDB, you can download, install, and have YugabyteDB up and running within minutes by following the steps in [Quick start](../../../../quick-start/). Alternatively, you can use [Yugabyte Cloud](http://cloud.yugabyte.com/register), to get a fully managed database-as-a-service (DBaaS) for YugabyteDB.
+
+- [Go version 1.8](https://golang.org/dl/), or later, is installed.
+
+### SSL/TLS configuration
+
+Refer to the [SSL/TLS configuration](../../../../secure/tls-encryption/client-to-server/) page to launch your YugabyteDB cluster with SSL/TLS enabled. This is already taken care of for you in Yugabyte Cloud, as the cluster is pre-configured to do client-side authentication with SSL/TLS enabled.
```

Comment:
On a second thought, this may be re-written as
"This is already taken care of for you in Yugabyte Cloud, as the cluster is pre-configured with SSL/TLS enabling client-side authentication."
Client can select what level of SSL/TLS to use, cluster only enables it.
Applies for the other two docs as well.

Reply:
Thank you. I will add this change.